README.md

```diff
@@ -212,8 +212,10 @@ const MIN_CONSTANT: i32 = 0;
 
 ```
 
-### must_match
-Tests whether the 2 fields are equal. `must_match` takes 1 string argument. It will error if the field
+### must_match and must_not_match
```

Comment:
I'm wondering if we can use `must_match` rather than create a new validator. Something like `must_match(expected=false)` with a default of true (bad name but just to illustrate)

Reply:
As you didn't approve generic `not()` ([Add not_match validator #141](https://github.com/Keats/validator/issues/141#issuecomment-766959104)) and we have `contains` and `does_not_contain` I thought that we need a new one

Reply:
So, can we move on?